leetcode/src/7.c

```diff
@@ -0,0 +1,35 @@
+int reverse(int x){
```

Comment:
So many checking here. I have a good solution for it:
```
int reverse(int x){
    long res = 0;
    while (x != 0)
    {
        int tail = x % 10;
        res = res * 10 + tail;
        /* Check overflow */
        if (res > INT_MAX || res < INT_MIN)
            return 0;
        x = x / 10;
    }
    return (int)res;
}
```

Reply:
Here you are using long int. What if the system we are using, supports up to 32-bit? What if the OS does not support long int?

Reply:
The question is not limit only for 32-bit.

Reply:
Yeah if we don't put that restriction then it will work. So do you want me to change and re-push?

Reply:
You can put the comment on your way, it works only for 32-bit system. And then adding my solution below. Thank you

Reply:
Committed the changes. Thanks